### PR TITLE
Added dir attribute to html tag

### PR DESCRIPTION
**Pull request by blakeganderson**
**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

This PR should be merged in AFTER https://github.com/WPMedia/arc-themes-feature-pack/pull/434 , which updates the feature pack blocks.json
## Jira Ticket

- [THEMES-1181](https://arcpublishing.atlassian.net/browse/THEMES-1181)

## Acceptance Criteria

The text direction value can map directly to the dir attribute on the <html> tag in our default output type block.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout branch THEMES-1181 in both the blocks and feature pack repos `git checkout 1181`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block`
3. Open a local page, such as: http://localhost/pf/2022/08/11/on-oregons-central-coast-a-low-key-exploration-of-wineries-restaurants-and-nature/?_website=the-gazette
4. Open the dev tools and navigate to the elements tab
5. Verify that the HTML tag pulls in the textDirection value from the feature pack blocks.json

### Before

HTML tag did not include a "dir" attribute
image

### After

HTML tag includes the "dir" attribute which pulls in the value set in the feature pack blocks.json
image

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1181]: https://arcpublishing.atlassian.net/browse/THEMES-1181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/default-output-block/output-types/default.jsx [hunk 1]
```diff
@@ -106,6 +106,7 @@ const SampleOutputType = ({
 		querylyId,
 		querylyOrg,
 		locale,
+		textDirection,
```
Comment (vgalatro) [inline blocks/default-output-block/output-types/default.jsx]: We should have this default to `ltr`, so we don't end up with an empty `dir` attribute.

**Review by vgalatro — CHANGES_REQUESTED** [review]
Just a tiny change needed, this is looking good though!

File: blocks/default-output-block/output-types/default.jsx (hunk 1 above)
Comment (blakeganderson) [reply blocks/default-output-block/output-types/default.jsx]: good catch, I went ahead and added the default as well as a test to check for the default when no textDirection is passed in

**Review by vgalatro — APPROVED** [review]
LGTM!

**Pull request merged**